ngraph/core/src/op/experimental_detectron_detection_output.cpp

```diff
@@ -0,0 +1,139 @@
+//*****************************************************************************
+// Copyright 2017-2020 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+
+#include <memory>
+
+#include "itt.hpp"
+#include "ngraph/attribute_visitor.hpp"
+#include "ngraph/op/experimental_detectron_detection_output.hpp"
+#include "ngraph/runtime/host_tensor.hpp"
+
+using namespace std;
+using namespace ngraph;
+
+NGRAPH_RTTI_DEFINITION(op::v6::ExperimentalDetectronDetectionOutput,
+                       "ExperimentalDetectronDetectionOutput",
+                       6);
+
+op::v6::ExperimentalDetectronDetectionOutput::ExperimentalDetectronDetectionOutput(
+    const Output<Node>& input_rois,
+    const Output<Node>& input_deltas,
+    const Output<Node>& input_scores,
+    const Output<Node>& input_im_info,
+    const Attributes& attrs)
+    : Op({input_rois, input_deltas, input_scores, input_im_info})
+    , m_attrs(attrs)
+{
+    constructor_validate_and_infer_types();
+}
+
+bool op::v6::ExperimentalDetectronDetectionOutput::visit_attributes(AttributeVisitor& visitor)
+{
+    visitor.on_attribute("score_threshold", m_attrs.score_threshold);
```

Comment:
Please, add `NGRAPH_OP_SCOPE` macros like in the https://github.com/openvinotoolkit/openvino/pull/3730

Reply:
Done.